apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ServiceAccountChecklist.spec.tsx

```diff
@@ -0,0 +1,148 @@
+import { act, render, screen } from '@testing-library/react';
+import ServiceAccountChecklist from 'components/config-screen/api-access/display/ServiceAccountChecklist';
+import { ERROR_TYPE_MAP } from 'apis/apiTypes';
+
+const invalidServiceAccountError = {
+  errorType: ERROR_TYPE_MAP.invalidServiceAccount,
+  message: 'Invalid Service Account Error Message.',
+  status: 500,
+};
+const adminApiError = {
+  errorType: ERROR_TYPE_MAP.disabledAdminApi,
+  message: 'Admin API Error Message.',
+  status: 500,
+};
+
+const dataApiError = {
+  errorType: ERROR_TYPE_MAP.disabledDataApi,
+  message: 'Data API Error Message.',
+  status: 500,
+};
+
+const ga4PropertiesError = {
+  errorType: ERROR_TYPE_MAP.noAccountsOrPropertiesFound,
+  message: 'No GA4 Properties Error Message.',
+  status: 500,
+};
+
+describe('Service Account Checklist', () => {
+  it('is the active happy path', async () => {
+    await act(async () => {
+      render(
+        <ServiceAccountChecklist
+          invalidServiceAccountError={undefined}
+          adminApiError={undefined}
+          dataApiError={undefined}
+          ga4PropertiesError={undefined}
+          parameters={{ propertyId: '123' }}
+        />
+      );
+    });
+
+    await screen.findByText('Service Account');
+    await screen.findByText('Admin api');
+    await screen.findByText('Data api');
+    await screen.findByText('GA4 Account Properties');
+    await screen.findAllByText(/Success!/);
```

Comment:
Do you need all of the `awaits` here, or just one? Additionally I am noticing in this file there are not assertions being made like `expect().toBe()`, is there a reason for that?

Reply:
Removed the awaits. I didn't do any `expect` calls because I don't know what else to expect other than these checklist statuses being rendered which is already checked by calling `findBy`

Reply:
ah okay I am following your logic, do the tests ever fail when you do not have assertions? or does it just timeout? I am just curious now 😄

Reply:
Nope, everything seems to be in a happy path